src/client/App.js

```diff
@@ -2,14 +2,31 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 
 import { Home } from './containers/Home/Home';
+import SignIn from './containers/SignIn';
+import SignUp from './containers/SignUp';
+import ResetPassword from './containers/ResetPassword';
+import PrivateRoute from './components/Auth/PrivateRoute';
+import { useAuth } from './hooks/useAuth';
+import Header from './components/Navigation/Header';
+import Profile from './containers/Profile';
+import Loader from './components/Loader';
 
 function App() {
+  const { isAuth, isLoading } = useAuth();
+  if (isLoading) return <Loader />;
   return (
     <Router>
+      <Header isAuth={isAuth} />
```

Comment:
Same for all `isAuth` in the PR
```suggestion
      <Header isAuthenticated={isAuthenticated} />
```